README.rst

```diff
@@ -21,6 +21,7 @@ We provide Python wheels (prebuilt binaries) for OSX 64 bits and Linux 64 bits:
 
 .. code-block:: bash
 
+    $ pip install wheel
```

Comment:
>Let's add in the README, that to run the tests you need to use USE_LOCAL_WOLFSSL=<path/to/wolfssl> pytest if building with a non-standard wolfSSL location.

Still missing this in the tests section. After that, should be good. Nice PR.

Reply:
Woops, my bad. Fixing now.